Sources/PointFree/Episode/Show.swift

```diff
@@ -338,7 +334,7 @@ private let downloadsView = View<String> { codeSampleDirectory -> [Node] in
   guard !codeSampleDirectory.isEmpty else { return [] }
 
   return [
-    div([Styleguide.class([Class.padding([.mobile: [.leftRight: 3], .desktop: [.leftRight: 4]])])],
+    div([Styleguide.class([Class.padding([.mobile: [.leftRight: 3], .desktop: [.leftRight: 4]]), Class.padding([.mobile: [.bottom: 3]])])],
```

Comment:
screenshot test caught a regression here where the credits module had extra top padding to make up for the fact that the downloads maybe be absent. easy enough to fix